src/resources/Resource.ts

```diff
@@ -1,16 +1,11 @@
-export default abstract class Resource<TContext> {
+import { Maybe, Obj } from '../utils/typing';
+
+export default abstract class Resource<TContext = any> {
   context: TContext;
 
   constructor(context: TContext) {
     this.context = context;
   }
 
-  abstract get<T>(
-    _id: string,
-  ): (T | null | undefined) | Promise<T | null | undefined>;
-
-  // TODO: Once we have a concept of a model, this should go on there.
-  makeId({ id }: any): string {
```

Comment:
why did we delete this?

Reply:
this is now on Node, so that it can use TSource from the node